### PR TITLE
Replaced JShint with Eslint (Closes #6)

### LINKED ISSUE #6: Replace jshint with eslint

### PR DESCRIPTION
**Pull request by alallier**
Replaced JShint with Eslint and fixed accompanying errors. (Closes #6)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +64,6 @@
   "_from": "mkroosevelt@*",
   "gitHead": "",
   "scripts": {
-    "test": "jshint mkroosevelt.js sampleApp/app.js sampleApp/mvc/controllers/404.js sampleApp/mvc/controllers/homepage.js sampleApp/mvc/controllers/robots.txt.js sampleApp/mvc/models/global.js"
+    "lint": "eslint ."
```
Comment (kethinov) [inline package.json]: add this to devDependencies

Comment (kethinov) [inline package.json]: my bad i meant to say version eslint to a specific version

**Comment by alallier** [comment]
Updated with versioned Eslint. It's good to note that this is still [issue](https://github.com/eslint/eslint/issues/3352) is still a problem. Do you still want the most current version of Eslint to go into the project? The same will apply to #29.

**Comment by alallier** [comment]
This should be all set assuming you take into account the above comment.

**Pull request merged**